middleware.js

```diff
@@ -285,15 +298,15 @@ module.exports = function(options) {
         }
 
         if (sassStats.mtime > cssStats.mtime) { // Source has changed, compile it
-          if (debug) { log('debug', 'modified', cssPath); }
+          log('debug', 'compile', cssPath, 'was modified');
```

Comment:
this implies the css was modified, not the scss; should the log have sassPath there instead?